pynecone/state.py

```diff
@@ -730,11 +735,18 @@ def get_delta(self) -> Delta:
         """
         delta = {}
 
-        # Return the dirty vars and dependent computed vars
-        self._mark_dirty_computed_vars()
-        delta_vars = self.dirty_vars.intersection(self.base_vars).union(
-            self._dirty_computed_vars()
+        # Apply dirty variables down into substates
+        self.dirty_vars.update(self._always_dirty_computed_vars())
```

Comment:
Maybe we can create a function like

```python
def get_dirty_vars(self):
    return {**self.dirty_vars, **self._always_dirty_computed_vars()}
```

and that way we never have to add the always dirty vars to the vars that get cleaned up